scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/package.scala

```diff
@@ -150,7 +150,7 @@ package object sparkey extends SparkeyReaderInstances with SparkeyCoders {
     def typedSparkeySideInput[T](
       basePath: String,
       decoder: Array[Byte] => T,
-      cache: Cache[String, T] = null
+      cache: Cache[String, T] = Cache.noOp[String, T]
```

Comment:
To be more in line with beam and other IOs, we've almost always used `null` as default, handled internally later on.
Can you update the PR to stick to this style ?

Reply:
just pushed a change to add a null check within the method instead - is that in-line with that style? or would it be preferrable to handle a null `cache` in the `TypedSparkeyReader` class itself?